openml-datarobot/README.md

```diff
@@ -1,12 +1,12 @@
 # DataRobot API
 
-This module contains the classes and logic related with the import of a model generated in [DataRobot](https://www.datarobot.com/).
+This module contains the classes and logic that allow to import models generated in [DataRobot](https://www.datarobot.com/).
 
-The current implementation allows to import a binary classification model exported in a Jar file.
+You can use the current implementation to import a binary classification model exported in a Jar file.
```

Comment:
maybe out of scope of this PR, but what about non binary classification models? don't we support it? why not? shouldn't this limitation be explicit and possible justified in the README?

Reply:
Since we already clearly say what we do support, I think it's clear that multiclass is not supported. But if there is a strong reason why we should highlight it, we can do it. 
Or we can remove "binary" from this sentence and add another one to state it explicitly. But usually it's nicer to focus on what we do instead of what we don't.

I don't think we need to justify why it's not supported.